docs/develop/go/core-application.mdx

```diff
@@ -567,7 +439,9 @@ func (a *YourActivityObject) YourActivityDefinition(ctx context.Context, param Y
 }
 ```
 
-### How to customize Activity Type in Go {#customize-activity-type}
+### Customize Activity Type in Go {#customize-activity-type}
```

Comment:
You replaced "in Go" with "using the Go SDK" above so maybe you can replace this with 'using the Go SDK' as well

Reply:
+1 also Type/Types